### PR TITLE
CASMTRIAGE-6742 :  troubleshooting

### PR DESCRIPTION
**Pull request by Pankhuri-Rajesh**
# Description

Adding few troubleshooting steps for cfs error count. 

Resolves [CASMTRIAGE-6742](https://jira-pro.it.hpe.com:8443/browse/CASMTRIAGE-6742)

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/iuf/IUF.md [hunk 1]
```diff
@@ -838,6 +838,22 @@ The following actions may be useful if errors are encountered when executing `iu
   of this document includes links to descriptions of each stage. Each of those descriptions includes an **Execution Details** section describing how to find the appropriate code in the IUF `workflows` directory to understand the
   workflow and debug the issue.
 
+### 4. Specific scenarios
+
+ 1. IUF workflow loops while rebuilding a management node.
+
+    - IUF loops while waiting for CFS to complete configuration of a management node. This step might not be completing because the CFS error count for the node has exceeded the   maximum retry count for applying the configuration.
```
Comment (rajeshranganathan-hpe) [inline operations/iuf/IUF.md]: IUF workflow may loop while waiting...

the   maximum (remove extra space)

File: operations/iuf/IUF.md [hunk 2]
```diff
@@ -838,6 +838,22 @@ The following actions may be useful if errors are encountered when executing `iu
...
+    - Look at the Ansible logs for the CFS configuration operation for that node and attempt to rectify the problem.
+    - After resolving the problem, update the default error count in CFS using the below command.
+
+         ```bash
+         ncn# cray cfs components update --enabled true --state '[]' --error-count 0 --format json $XNAME
+         ```
+
+    - Once the error count is reset, the CFS batcher pod will restart configuration for the node. If it does not start within a few minutes,
```
Comment (rajeshranganathan-hpe) [inline operations/iuf/IUF.md]: CFS batcher pod - replace with just CFS

File: operations/iuf/IUF.md [hunk 3]
```diff
@@ -838,6 +838,22 @@ The following actions may be useful if errors are encountered when executing `iu
...
+   check the CFS batcher pod log to determine whether CFS batcher is doing back off logic due to too many failures from CFS batches it has started.
```
Comment (rajeshranganathan-hpe) [inline operations/iuf/IUF.md]: check the CFS batcher pod log to determine whether CFS batcher is doing back off logic due to too many failures from CFS batches it has started.

Can be replaced with

check whether CFS is unable to start the configuration again for the node due to any other issue. Rectify the problem by referring to the CFS trouble shooting guide. [https://github.com/Cray-HPE/docs-csm/blob/18b44e940acf086abd9e6ab60f34455ce5125c97/operations/configuration_management/Troubleshoot_CFS_Sessions_Failing_to_Start.md]

**Review by rajeshranganathan-hpe — APPROVED** [review]

File: operations/iuf/IUF.md [hunk 4]
```diff
@@ -838,6 +838,22 @@ The following actions may be useful if errors are encountered when executing `iu
...
+
+ 1. IUF workflow may loop while rebuilding a management node.
+
+    - IUF loops while waiting for CFS to complete configuration of a management node. This step might not be completing because the CFS error count for the node has exceeded the maximum retry count for applying the configuration.
+    - Look at the Ansible logs for the CFS configuration operation for that node and attempt to rectify the problem.
+    - After resolving the problem, update the default error count in CFS using the below command.
+
+         ```bash
+         ncn# cray cfs components update --enabled true --state '[]' --error-count 0 --format json $XNAME
+         ```
```
Comment (leliasen-hpe) [inline operations/iuf/IUF.md]:
```suggestion
    - After resolving the problem, update the default error count in CFS using the below command. Run this command form a master or worker node. Set environment variable `XNAME` to be the xname of the node where the CFS configuration has failed.

         ```bash
         cray cfs components update --enabled true --state '[]' --error-count 0 --format json $XNAME
         ```
```

**Review by leliasen-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**